src/native/external/cgmanifest.json

```diff
@@ -46,7 +46,7 @@
         "Type": "git",
         "Git": {
           "RepositoryUrl": "https://github.com/zlib-ng/zlib-ng",
```

Comment:
BTW: I have noticed that we do not have the latest brotli release either. Do we plan to update brotli too? (unrelated to this PR)

Reply:
I can look into this and add it to my backlog. cc @ericstj

Reply:
Opened an issue to track this https://github.com/dotnet/runtime/issues/106887